packages/commonwealth/client/scripts/views/pages/user_dashboard/index.tsx

```diff
@@ -63,7 +67,40 @@ const UserDashboard = (props: UserDashboardProps) => {
       ? DashboardViews.ForYou
       : DashboardViews.Global;
 
-  React.useEffect(() => {
+  // This hook logic adds sticky behavior to the tabs on this page when shouldAddStickyTabs is true
+  useNecessaryEffect(() => {
```

Comment:
this can be turned into a reuseable hook, if we want to do this sticky behaviour elsewhere

Reply:
even thought this is used only in once place, we should move it to the hook anyways

Reply:
done